lib/OpenQA/BuildResults.pm

```diff
@@ -138,6 +138,8 @@ sub compute_build_results {
     }
     if ($tags) {
         # caveat: A tag that references only a build, not including a version, might be ambiguous
+        use Data::Dumper;
```

Comment:
It was still WIP (because the check for the version was missing). Removed it now.